### DISCUSSION TITLE
More IDE style like React-Grid-Layout

### DISCUSSION THREAD

**Discussion by hartum** [jbaysolutions/vue-grid-layout#517] [Ideas]
Hi, first of all great great job.

In description you wrote _"Heavily inspired by React-Grid-Layout"_ but originally this React script is more like a IDE with tabs, docking panels, etc.. and vue-grid-layout is more like a 'supervitamined mansonry' (take it as a compliment, not an insult).

Is there any plan to make a script more like the original React-Grid-Layout? Is there any plan to make a script more like the original? even if you create it as a new project?

The closest thing I've found so far are these scripts:
- [vue-golden-layout](https://github.com/emedware/vue-golden-layout)
- [jQwidget component](https://www.jqwidgets.com/vue/vue-dockinglayout/#https://www.jqwidgets.com/vue/vue-dockinglayout/vue-dockinglayout-fluidsize.htm)
- [lumino implementation for vue](https://kinoshita.eti.br/2020/03/28/using-lumino-with-vue.html)
- [smartHTML elements](https://www.htmlelements.com/vue/demos/docking-layout/overview/)

none of them are developed natively in vue, and three of them are part of a larger package. Only vue-golden-layout has a clear approach.

So, what do you think of the idea of creating something similar?

**Comment by gmsa** [comment]
I'm sorry, are you talking about this [react-grid-layout](https://github.com/STRML/react-grid-layout)? Am I missing something?

**Reply by hartum** [reply]
yes, just that

**Reply by gmsa** [reply]
how is that like an IDE? Trust me, vue-grid-layout is based on react-grid-layout

**Reply by hartum** [reply]
Upssss, I think it is a big mistake by my side, I just cross the names, let me check

**Reply by hartum** [reply]
you are 100% right, I am wrong. I was talking about [flexlayout-react](https://github.com/caplin/FlexLayout#readme) and at some point in my mind I crossed the two names.

I apologize
